### PR TITLE
Feat/android fastlane

### PR DESCRIPTION
**Pull request by lavi-moolchandani**

### CODE REVIEW

File: example/android/fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    f.close
+    build_android_app(
+      task: 'assemble',
+      build_type: 'Release'
+    )
+    commit = last_git_commit
+    firebase_app_distribution(
+      app: "1:408505141940:android:7fae09bd993f283fc325e1",
+      testers: "internal, external",
+      android_artifact_path: "/Users/lavi/Sites/projects/HMS/100ms-react-native/example/android/app/build/outputs/apk/release/app-release.apk",
```
Comment (ygit) [inline example/android/fastlane/Fastfile:39]: path-name would be generic

File: example/android/fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    f.write(s)
+    f.close
+    build_android_app(
+      task: 'assemble',
+      build_type: 'Release'
+    )
+    commit = last_git_commit
+    firebase_app_distribution(
+      app: "1:408505141940:android:7fae09bd993f283fc325e1",
+      testers: "internal, external",
```
Comment (ygit) [inline example/android/fastlane/Fastfile:38]: `testers: "internal", "External",`
"E" in caps

File: example/android/fastlane/Fastfile (hunk 1 above)
Comment (ygit) [reply example/android/fastlane/Fastfile:39]: `android_artifact_path: "../build/app/outputs/apk/release/app-release.apk",`

**Comment by lavi-moolchandani** [comment]
Creating new branch with fastlane android + ios

**Pull request closed without merging**